ui/src/app/pages/workspace/workspace-edit-section.tsx

```diff
@@ -0,0 +1,86 @@
+import {FlexRow} from 'app/components/flex';
+import {InfoIcon} from 'app/components/icons';
+import {TooltipTrigger} from 'app/components/popups';
+import colors from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import * as React from 'react';
+
+export const styles = reactStyles({
+  header: {
+    fontWeight: 600,
+    lineHeight: '24px',
+    color: colors.primary
+  },
+  infoIcon: {
+    height: '16px',
+    marginLeft: '0.2rem',
+    width: '16px'
+  },
+  requiredText: {
+    fontSize: '13px',
+    fontStyle: 'italic',
+    fontWeight: 400,
+    color: colors.primary,
+    marginLeft: '0.2rem'
+  },
+  text: {
+    fontSize: '13px',
+    color: colors.primary,
+    fontWeight: 400,
+    lineHeight: '24px'
+  }
+});
+
+interface Props {
+  // As child description or header can be either string or reactNode
+  children?: any;
+  description?: any;
+  header: any;
+  index?: string;
+  indent?: boolean;
+  largeHeader?: any;
+  required?: boolean;
+  tooltip?: React.ReactNode;
+  subHeader?: string;
+  style?: any;
```

Comment:
Not sure the datatype for style

Reply:
Nothing passes a style prop in to this component. Just get rid of the prop.

Reply:
We cannot since WorkspaceEdit has a common style marginBottom and then add the specific style from each section thats using it in workspace-edit

Reply:
Looking at it, it looks to me like there are a number of existing usages that pass that in as well, like here: https://github.com/all-of-us/workbench/blob/nsaxena/warningResearchPurpose/ui/src/app/pages/workspace/workspace-edit.tsx#L778

I have one small thing here, but its probably out of scope for this ticket. Now that we have more files relating to workspace edit, can you make a tech debt ticket to move them to their own folder?

Reply:
They do however since almost none defines it as interface not sure if i can leave style type as any.  For description and header it makes sense sine it can be either string or node. So now the question is are we OK with data type of style being any?

Reply:
`any` is probably fine, or maybe `React.CSSProperty`?

Reply:
Whatever type our `reactStyle()` method returns.

Reply:
React.CSSProperty gives the error 
Namespace 'React' has no exported member 'CSSProperty'

So i guess i'll leave it as ANY

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/RW-4612 to move it to a new folder

Reply:
My bad on missing that the style prop was used, must not have expanded enough things.